Sources/MongoSwift/MongoCollection+Indexes.swift

```diff
@@ -258,35 +258,33 @@ extension MongoCollection {
             throw getErrorFromReply(bsonError: error, from: reply)
         }
 
-        return forModels.map { $0.options?.name ?? $0.defaultName }
+        return models.map { $0.options?.name ?? $0.defaultName }
     }
 
     /**
      * Drops a single index from the collection by the index name.
      *
      * - Parameters:
      *   - name: The name of the index to drop
-     *   - writeConcern: An optional WriteConcern to use for the command
+     *   - options: Optional `DropIndexOptions` to use for the command
      *
      * - Throws:
+     *   - `ServerError.writeError` if an error occurs while performing the command.
```

Comment:
@patrickfreed, these errors were listed for the other drop methods, so I duplicated them here as well.

Reply:
It was originally omitted since we were using the mongoc drop index helper (which doesn't return write errors, I think). Now that we're using the generic write command function, write errors are possible.